.github/workflows/release.yml

```diff
@@ -89,6 +89,55 @@ jobs:
           config: .github/config/slack.yml
         if: contains(toJson(github.event.commits.*.message), 'BREAKING CHANGE')
 
+      # Step 12: Merge prod branch into main
+      - name: Create PR to merge prod into main
+        id: createpr
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const ownerAndRepo = '${{ github.repository }}';
+            const [owner, repo] = ownerAndRepo.split('/');
+            const head = "prod";
+            const base = "main";
+            const title = "feat: Merge prod into main";
+            const pr = await github.rest.pulls.list({owner, repo, base, head: `${owner}:${head}`})
+            if (pr.data.length>0) {
+              return pr.data[0].number;
+            }
+            const result = await github.rest.pulls.create({
+              owner,
+              repo,
+              head,
+              base,
+              title,
+            });
+            return result.data.number;
+
+      # Step 13: Automerge PR
+      - name: Enable Automerge PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: peter-evans/enable-pull-request-automerge@v3
+        with:
+          pull-request-number: ${{steps.createpr.outputs.result}}
+          merge-method: squash
+      
+      # Step 14: Generate token for autoapproving PR
+      - name: Generate token
+        id: generate_token
+        uses: chanzuckerberg/github-app-token@v1.1.4
+        with:
+          app_id: ${{ secrets.CZI_GITHUB_HELPER_APP_ID }}
+          private_key: ${{ secrets.CZI_GITHUB_HELPER_PK }}
+
+      # Step 15: Autoapprove PR
+      - name: Autoapprove PR
```

Comment:
Oh 😯
I had no idea we could auto-approve the PRs ⭐️
This is fantastic! Thanks for sharing 🥳